e2e_test/source_inline/kafka/shared_source.slt

```diff
@@ -230,6 +230,87 @@ internal_table.mjs --name s0 --type source
 3,"{""split_info"": {""partition"": 3, ""start_offset"": 11, ""stop_offset"": null, ""topic"": ""shared_source""}, ""split_type"": ""kafka""}"
 
 
+# # Note: the parallelism depends on the risedev profile.
+# # So scale tests below are commented out.
```

Comment:
I think these unenforced tests can be stale quickly. Shall we remove this as we've implemented madsim tests?